app/models/mixins/ui_service_mixin.rb

```diff
@@ -0,0 +1,24 @@
+module UiServiceMixin
```

Comment:
It's weird that it's a mixin for services, but lives in app/models, but considering we don't have a services/mixins directory right now, this is fine... cc @martinpovolny